addons/mass_mailing/static/src/js/wysiwyg.js

```diff
@@ -20,6 +20,29 @@ const MassMailingWysiwyg = Wysiwyg.extend({
         return res;
     },
 
+    /**
+     * @override
+     */
+     getValue: function () {
+        let editable = this._super(...arguments);
+        const parsedDocument = new DOMParser().parseFromString(editable, "text/html");
```

Comment:
`parsedDocument` is a very generic name. Literally any document is some kind of parsed document... I'd call it `editable` but it might be confused for the _real_ editable whereas this one is merely a clone parsed from the string. Maybe `parsedValue` ?